nats-server/connz.yml

```diff
@@ -18,3 +18,45 @@ observation:
     tag:
       - name: nats.server.id
         value: json:/connz $.server_id
+
+  - name: connz_conn
+      metricNamespace: nats
+      path: $.connections[?(@.name=${connectionName}})]
+
+      metric:
+        - name: server.connections.bytes.pending
```

Comment:
since these are per connection metric should the name be `server.connection.bytes.pending`?

Reply:
it have bytes.in, bytes.out so i use name bytes.pending. Should i change to just server.connection.bytes.pending

Reply:
I change to server.connection.pending